curriculum/challenges/english/08-coding-interview-prep/data-structures/delete-a-node-with-two-children-in-a-binary-search-tree.english.md

```diff
@@ -23,23 +23,23 @@ Let's finish our <code>remove</code> method by handling the third case. We've pr
-  - text: The <code>remove</code> method removes leaf nodes from the tree
+  - text: The <code>remove</code> method should remove leaf nodes from the tree.
     testString: "assert((function() { var test = false; if (typeof BinarySearchTree !== 'undefined') { test = new BinarySearchTree() } else { return false; }; test.add(5); test.add(3); test.add(7); test.add(6); test.add(10); test.add(12); test.remove(3); test.remove(12); test.remove(10); return (typeof test.remove == 'function') ? (test.inorder().join('') == '567') : false})());"
-  - text: The <code>remove</code> method removes nodes with one child.
+  - text: The <code>remove</code> method should remove nodes with one child.
     testString: assert((function() { var test = false; if (typeof BinarySearchTree !== 'undefined') { test = new BinarySearchTree() } else { return false; }; if (typeof test.remove !== 'function') { return false; }; test.add(-1); test.add(3); test.add(7); test.add(16); test.remove(16); test.remove(7); test.remove(3); return (test.inorder().join('') == '-1'); })());
-  - text: Removing the root in a tree with two nodes sets the second to be the root.
+  - text: Removing the root in a tree with two nodes should set the second to be the root.
     testString: assert((function() { var test = false; if (typeof BinarySearchTree !== 'undefined') { test = new BinarySearchTree() } else { return false; }; if (typeof test.remove !== 'function') { return false; }; test.add(15); test.add(27); test.remove(15); return (test.inorder().join('') == '27'); })());
-  - text: The <code>remove</code> method removes nodes with two children while maintaining the binary search tree structure.
+  - text: The <code>remove</code> method should removes nodes with two children while maintaining the binary search tree structure.
     testString: assert((function() { var test = false; if (typeof BinarySearchTree !== 'undefined') { test = new BinarySearchTree() } else { return false; }; if (typeof test.remove !== 'function') { return false; }; test.add(1); test.add(4); test.add(3); test.add(7); test.add(9); test.add(11); test.add(14); test.add(15); test.add(19); test.add(50); test.remove(9); if (!test.isBinarySearchTree()) { return false; }; test.remove(11); if (!test.isBinarySearchTree()) { return false; }; test.remove(14); if (!test.isBinarySearchTree()) { return false; }; test.remove(19); if (!test.isBinarySearchTree()) { return false; }; test.remove(3); if (!test.isBinarySearchTree()) { return false; }; test.remove(50); if (!test.isBinarySearchTree()) { return false; }; test.remove(15); if (!test.isBinarySearchTree()) { return false; }; return (test.inorder().join('') == '147'); })());
-  - text: The root can be removed on a tree of three nodes.
+  - text: The root should be able to removable on a tree of three nodes.
```

Comment:
```suggestion
  - text: The root should be able to be removed on a tree of three nodes.
```

Reply:
@moT01 I think what I meant to change it to was:
> The root should be removable on a tree of three nodes.

If you are ok with that, can you change the suggestion?

Reply:
✔️